### ISSUE TITLE
Error when installing mlocate

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#669]
`mkdir: cannot create directory ‘/usr/local/var/mlocate’: Permission denied`

The residual effect of the above error:
```
$ sudo updatedb
updatedb: can not open a temporary file for `/usr/local/var/mlocate/mlocate.db'
```

**Comment by lyxell** [comment]
Looks like `/usr/local/var` isn't owned by `chronos`. Could you check the permissions via `ls -la`?

**Comment by lyxell** [comment]
I think https://github.com/skycocker/chromebrew/blob/4a1ea3fb08ffe753193d874ac04700df07419c1b/packages/mlocate.rb#L9 is supposed to fix this issue but it looks like it's not working.

**Comment by uberhacker** [comment]
@lyxell: It appears /usr/local/var is a symlink:
```
$ ls -la /usr/local/var
lrwxrwxrwx 1 chronos chronos 4 Dec 15 09:20 /usr/local/var -> /var
```
Ownership on /var
```
drwxr-xr-x   9 root root  4096 May 21 09:59 var
```

**Comment by lyxell** [comment]
That's problematic, do you think you can figure out what package created the symlink?

**Comment by uberhacker** [comment]
PR #686 should fix this issue.

**Comment by jam7** [comment]
I think all of us don't have this problem since we have `/usr/local/var` directory instead of a symbolic link.  I guess you might crate it while you were working on mandb package.

**Comment by uberhacker** [comment]
@jam7: I'm getting this error on CloudReady.  The symlink is created internally, I believe.  In any event, we shouldn't assume that we'll always have a directory instead of a symlink but should have some process that corrects this situation.

**Comment by jam7** [comment]
Ah CloudReady.  That's make sense.  It may cause another problem since glibc223 installs a file at `/usr/local/var/db/Makefile`.  Didn't you have such problems?

I'll try CouldReady later.  I had one, but it stops working after upgrade.  ;)

**Comment by uberhacker** [comment]
Yeah, CloudReady was the source of other issues as well.  It killed my dual boot into Linux last time I installed it. :)

**Comment by uberhacker** [comment]
Fixed with #686.
